internal/tests/api/with-ns-registered/minipool_test.go

```diff
@@ -60,6 +60,85 @@ func TestMinipoolDepositAndStake(t *testing.T) {
 	defer nodeset_cleanup(snapshotName)
 
 	depositAndStakeMinipool(t)
+	simulateEthRewardToYieldDistributor(t)
+}
+
+// Utility function to send ETH and advance blockchain time
+func sendEthAndAdvanceTime(t *testing.T, address common.Address, slotsToAdvance int) {
```

Comment:
If this is a utility function then I'd suggest making an `amount` argument and setting the Value in the opts to that instead of hardcoding 1e18.

Reply:
fixed